sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -810,6 +810,26 @@ case class LocalLimit(limitExpr: Expression, child: LogicalPlan) extends OrderPr
   }
 }
 
+/**
+ * This is similar with [[Limit]] except,
+ *
+ * - It does not have plans for global/local separately because currently there is only one
+ *   implementation which does both global/local tails. See
```

Comment:
I think there is no global and local because it is always be wrapped by `ReturnAnswer`?

Reply:
Ah hm, I was trying to explain conceptually - https://github.com/apache/spark/pull/26809/files#diff-379cccace8699ca00b76ff5631222adeR85-R111
I think it will still do a global limit-ish operation in single partition

Reply:
Let me try to reword it